seeds: add violet subdomain dummy data

yashkir opened the pull request:
resolves #628 

- seed to look like violet.restarone.solutions
- add faker data

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+
+if Apartment::Tenant.current == 'violet'
+  user_orange = User.create!(
+    email: 'orange@rails.com', 
```
please use `Faker::Internet.email `

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+if Apartment::Tenant.current == 'violet'
+  user_orange = User.create!(
+    email: 'orange@rails.com', 
+    name: 'Orange Tester',
```
please use `Faker::Name.name`

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+  user_red.update!(User::FULL_PERMISSIONS)
+
+  user_blue = User.create!(
+    email: 'blue@rails.com', 
```
same as same as https://github.com/restarone/violet_rails/pull/874/files#r913738056

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+
+  user_blue = User.create!(
+    email: 'blue@rails.com', 
+    name: 'Blue Tester',
```
same as https://github.com/restarone/violet_rails/pull/874/files#r913739531

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+    record: blog_post,
+    tag: 'wysiwyg',
+    content: "
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quae duo sunt, unum facit. Uterque enim summo bono fruitur,
```
please use `Faker::GreekPhilosophers.quote ` between paragraph/markup tags

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+
+  # Forum
+  # -----
+  ForumCategory.create!(name: "cats", slug: "cats2", color: "#0000CC")
```
please use `Faker::Educator.subject`

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+  ForumCategory.create!(name: "cats", slug: "cats2", color: "#0000CC")
+  ForumCategory.create!(name: "dogs", slug: "dogs2", color: "#00CCCC")
+
+  thread = user.forum_threads.create!(title: "Cats are great?", forum_category: ForumCategory.first)
```
please use `Faker::Educator.course_name`

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+
+  # Emails
+  # ------
+  email_thread = MessageThread.create!(recipients: ['violet@rails.com', 'red@rails.com'] )
```
please use `Faker::Internet.email `

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+  # Emails
+  # ------
+  email_thread = MessageThread.create!(recipients: ['violet@rails.com', 'red@rails.com'] )
+  email_thread.messages.create(content: "An invitation to a meeting. Please come here at 4pm on Friday.")
```
@yashkir this is a rich text field. you should use some markup and use `Faker::Movie.quote` to populate it

donrestarone commented on db/seeds.rb:
```diff
@@ -6,4 +9,131 @@
+  email_thread.messages.create(content: "Have to skip this one.", from: 'red@rails.com')
+
+  email_thread = MessageThread.create!(recipients: ['violet@rails.com'] )
+  email_thread.messages.create(content: "I really like your site!", from: 'violet@rails.com')
```
please use `Faker::Movie.quote `

donrestarone commented:
Great first effort @yashkir -- lets use Faker along with the other changes I suggested above. After you attach screenshots of the Freshly Seeded State :seedling: we can deploy this to `staging`

yashkir commented:
Added Faker and removed the subdomain. Should be fairly straightforward to extend these with fancier markup/date if needed:

![Screenshot_2022-07-09_22-59-48](https://user-images.githubusercontent.com/55562/178129754-43844f62-3a9c-4bed-bbc2-8f36acd25869.png)
![Screenshot_2022-07-09_23-00-03](https://user-images.githubusercontent.com/55562/178129751-abc98291-c9b3-4c57-8eb0-b0b602913125.png)
![Screenshot_2022-07-09_22-58-14](https://user-images.githubusercontent.com/55562/178129757-ac183b0a-5469-42a4-8413-eb0fb14f936e.png)
![Screenshot_2022-07-09_23-13-00](https://user-images.githubusercontent.com/55562/178129821-3fb12e7f-ade9-4af3-ba7a-75983593aaec.png)
![Screenshot_2022-07-09_22-57-45](https://user-images.githubusercontent.com/55562/178129758-4a834090-ad32-4b0f-b20f-866b4dccd3f2.png)
![Screenshot_2022-07-09_23-11-39](https://user-images.githubusercontent.com/55562/178129788-4eba4505-7edd-4e7d-881d-d66a94e376b5.png)
![Screenshot_2022-07-09_23-00-29](https://user-images.githubusercontent.com/55562/178129748-c6314832-37eb-4a0c-82e6-473e0c7ac4ad.png)

donrestarone commented on db/violet_seeds/violet.rb:
```diff
@@ -0,0 +1,1201 @@
+				<div class="d-flex align-items-center my-5">
+					<a href="/forum" class="btn btn-lg btn-primary btn-shadow">Forum</a>
+					<a href="https://github.com/restarone/violet_rails" class="d-flex align-items-center btn btn-lg font-weight-bold bg-transparent">
+					<img src="{{ cms:file_link 31 }}" class="img mr-2">
```
@yashkir where we use tokens like `{{ cms:file_link 31 }}` how are we ensuring these hard coded ID's (eg: `31`) match what is in the file system?

donrestarone commented:
@yashkir Faker looks good! Can you attach some pictures of the web pages? Eg: `violet.restarone.solutions` landing page

yashkir commented:
> @yashkir Faker looks good! Can you attach some pictures of the web pages? Eg: `violet.restarone.solutions` landing page

As mentioned in the review reply above, the images are not imported and show as blank. We could leave this as is, or import a minimal set (e.g. just the violet rails/restarone logos).

![Screenshot_2022-07-10_23-11-40](https://user-images.githubusercontent.com/55562/178181886-645f3445-840e-464d-819d-61c09dd0eb7f.png)
![Screenshot_2022-07-10_23-12-08](https://user-images.githubusercontent.com/55562/178181910-67bcbfb8-c724-445b-9522-30d4d11694f4.png)
![Screenshot_2022-07-10_23-12-23](https://user-images.githubusercontent.com/55562/178181915-dc39bbbb-3f4f-466e-b769-984792157617.png)

donrestarone commented:
@yashkir good call on it bloating source. But I think we should include it and strip it later (as part of a different task) during deployment. 

Can you go ahead and include the assets as well? So it looks just like `violet.restarone.solutions`

donrestarone commented:
@yashkir Since CI is passing and it looks like you have linked assets, I'm creating a base branch for you. Please use that branch for changes (if any) moving forward 👍
